shoes-core/spec/shoes/star_spec.rb

```diff
@@ -170,7 +170,7 @@
   end
 
   describe "redrawing region" do
-    subject { Shoes::Star.new(app, parent, 100, 100, 5, 50, 30, center: false, strokewidth: 0) }
+    subject { Shoes::Star.new(app, parent, 100, 100, 5, 50, 30, center: false) }
```

Comment:
as strokewidth now factors in directly, setting it here might help discover what value it is at? (I know there's another test that specifically tests it I thought it might be good for the general case)

Reply:
Yeah, that would make it more obvious. I think I was trying to show what the default does, but it makes more sense with the strokewidth's default value (1) called out.